bokehjs/examples/anscombe/anscombe.ts

```diff
@@ -0,0 +1,77 @@
+module Anscombe {
```

Comment:
The recommendation is to use `namespace` here instead of `module`. The `module` keyword is deprecated:
http://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html